File: SecondYear/Vaishnavi0-0_VaishnaviSingh_2226cseml1114_2/week_1/pattern9.java
import java.util.Scanner;

public class pattern9 {
    public static void main(String[] args) {
        Scanner read = new Scanner(System.in);
        int n = read.nextInt();
        printTriangle(n);
    }

    public static void printTriangle(int n) {
        // code here
        for (int i = n; i >= 1; i--) {
            for (int j = 1; j <= n; j++) {
                if (j <= i)
                    System.out.print("*");
                else
                    System.out.print(" ");
            }
            /*
             * for(int k=i;k<(n-i);k++)
             * {
             * System.out.print(" ");
             * }
             */
            for (int j = n; j >= 1; j--) {
                if (j <= i)
                    System.out.print("*");
                else
                    System.out.print(" ");
            }
            System.out.println();
        }
        for (int i = n; i >= 1; i--) {
            for (int j = n; j >= 1; j--) {
                if (j >= i)
                    System.out.print("*");
                else
                    System.out.print(" ");
            }
            for (int j = 1; j <= n; j++) {
                if (j >= i)
                    System.out.print("*");
                else
                    System.out.print(" ");
            }
            /*
             * for(int k=i;k<(n-i);k++)
             * {
             * System.out.print(" ");
             * }
             */

            System.out.println();
        }
    }

}

File: SecondYear/chaudharyaryanpanwar_AryanPanwar_2226cs1053_2/Week_1/Pattern9.cpp
class Solution {
    public :
    void printTriangle(int n){
        //code here
        for (int i = n ;i >= 1 ; i--){
            for (int j =1 ; j<=i;j++){
                cout <<"*";
            } 
            for (int j = 2*(n-i) ;j>0;j-- ){
                cout <<" ";
            }
            for (int j = i ; j>=1 ; j-- ){
                cout << "*";
            }
            cout << endl;
            //16 - 12 - 8- 4
        }
         for (int i = 1 ;i <= n ; i++){
            for (int j =1 ; j<=i;j++){
                cout <<"*";
            } 
            for (int j = 2*(n-i) ;j>0;j-- ){
                cout <<" ";
            }
            for (int j = i ; j>=1 ; j-- ){
                cout << "*";
            }
            cout << endl;
            //16 - 12 - 8- 4
        }
    }
};